2021-03-02 15:31:26,380   17700     INFO test_runner.py(154) - 预计并发线程数：2
2021-03-02 15:31:28,424   19652     INFO result.py(65) - 开始测试： test_rejected (test_login.test_saas.MyTest)
2021-03-02 15:31:28,425   19652     INFO test_saas.py(10) - test
2021-03-02 15:31:33,565   20668     INFO result.py(65) - 开始测试： test_login (test_login.test_login_demo.LoginDemo)
2021-03-02 15:31:33,567   20668    ERROR result.py(171) - 测试产生错误： test_login (test_login.test_login_demo.LoginDemo)
TypeError: test_login() missing 3 required positional arguments: 'case', 'username', and 'password'

2021-03-02 15:31:33,567   20668     INFO result.py(86) - 测试结束： test_login (test_login.test_login_demo.LoginDemo)
2021-03-02 15:31:33,567   20668     INFO result.py(87) - 耗时： 0.0019948482513427734
2021-03-02 15:31:38,912   19652     INFO test_saas.py(42) - 测试用例标题：双12活动打开1个宝箱
2021-03-02 15:31:39,240   19652     INFO HttpRequests.py(45) - post：https://pre.17kuxiu.com/user/getAnchorCoverAuditInfo
2021-03-02 15:31:39,240   19652     INFO HttpRequests.py(46) - 从请求到接口返回信息耗时：0.315643秒
2021-03-02 15:31:39,240   19652     INFO HttpRequests.py(26) - 接口返回数据：{"retCode":-100004,"retMsg":"头部信息不完整","success":false}
2021-03-02 15:31:39,244   19652    ERROR test_saas.py(52) - 失败：False is not true
2021-03-02 15:31:41,927   19652  WARNING result.py(186) - 测试未通过： test_rejected (test_login.test_saas.MyTest)
Traceback (most recent call last):
  File "D:\auto_saas_20210201\testcase\test_login\test_saas.py", line 49, in test_rejected
    self.assertTrue(r['retCode'] == 200)
AssertionError: False is not true

2021-03-02 15:31:41,928   19652     INFO result.py(86) - 测试结束： test_rejected (test_login.test_saas.MyTest)
2021-03-02 15:31:41,928   19652     INFO result.py(87) - 耗时： 13.501959323883057
2021-03-02 15:31:41,929   17700    ERROR test_runner.py(193) - 
Error	test_login (test_login.test_login_demo.LoginDemo)
Fail	test_rejected (test_login.test_saas.MyTest)

2021-03-02 15:31:41,932   17700     INFO test_runner.py(211) - 
测试结束！
运行时间: 0:00:17.339708
共计执行用例数量：2
执行成功用例数量：0
执行失败用例数量：1
跳过执行用例数量：0
产生异常用例数量：1

